### PR TITLE
Removing unused imports

### PR DESCRIPTION
**Pull request by gradientsky**
*Issue #, if available:*
https://github.com/awslabs/autogluon/issues/1927

*Description of changes:*
Removed unused imports

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: multimodal/src/autogluon/multimodal/data/trivial_augmenter.py [hunk 1]
```diff
@@ -6,8 +6,7 @@
 
 import random
 import logging
-from selectors import EpollSelector
```
Comment (gradientsky) [inline multimodal/src/autogluon/multimodal/data/trivial_augmenter.py:9]: This should address https://github.com/awslabs/autogluon/issues/1927

**Comment by szha** [comment]
Job PR-1928-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1928/2/index.html

**Review by zhiqiangdon — APPROVED** [review]
Thanks!

**Review by sxjscience — APPROVED** [review]

**Comment by sxjscience** [comment]
Just noticed that there are some conflicts. I feel that we can merge once the conflicts are resolved.

**Review by yinweisu — APPROVED** [review]

**Comment by sxjscience** [comment]
@gradientsky To address the MacOS issue, I submitted https://github.com/awslabs/autogluon/pull/1949 to remove `selectors`. Meanwhile, I think you can choose if you'd like to directly merge this PR or merge the other PR and then merge this PR.

**Pull request merged**